public/app/features/users/UsersActionBar.tsx

```diff
@@ -49,8 +49,8 @@ export const UsersActionBarUnconnected = ({
     { label: `Pending Invites (${pendingInvitesCount})`, value: 'invites' },
   ];
   const canAddToOrg: boolean = contextSrv.hasAccess(AccessControlAction.OrgUsersAdd, canInvite);
-  // backend rejects invitations if the login form is disabled
-  const showInviteButton: boolean = canAddToOrg && !config.disableLoginForm;
```

Comment:
yea we contemplated on using the externalUserMngInfo or just the disabledLoginForm. the cloud invite button is removed if the disabledLoginForm is true (since that is what clouds setting is set).

Do you know if the externalUserMngInfo is set in cloud?